controls/calendar/migrate-from-xamarin.md

```diff
@@ -6,13 +6,11 @@ position: 20
 slug: calendar-migrate-from-xamarin
 ---
 
-# Migrate from Xamarin.Forms Calendar to .NET MAUI Calendar
+# Migrating from Xamarin.Forms Calendar to .NET MAUI Calendar
 
-The Telerik UI for .NET MAUI Calendar control has been designed and built from the ground up as a completely new control with a new API and significant improvements over its Xamarin counterpart.
+The Telerik UI for .NET MAUI Calendar control has been designed and built from the ground up as a new control with a new API and significant improvements over its Xamarin counterpart.
 
-In Telerik UI for Xamarin, the Calendar also includes scheduling, while the Telerik UI for .NET MAUI Calendar and Scheduler are different controls. As of today, the development of the Scheduler is planned. 
-
->note The Telerik UI for .NET MAUI Calendar control provides selection and does not manage appointments like the Telerik Xamarin Calendar does. The appointments will be part of the Telerik .NET MAUI Scheduler control.  
+>important The Telerik UI for .NET MAUI Calendar control provides selection - single, multiple and range. The Telerik UI for .NET MAUI Calendar control does not manage appointments like the Telerik Xamarin Calendar does. The appointments will be part of the Telerik .NET MAUI Scheduler control. The Telerik UI for .NET MAUI Scheduler control is planned for R3 2023 release. 
```

Comment:
The first and the second sentence aren't logically connected. Why are they part of the same paragraph? You may need to add more context to keep them in the same paragraph. Do you mean that the Xamarin control doesn't support selection?
Instead of the note, consider this structure that is clearer and provides more details. If the information is wrong, change it.

```suggestion
When you plan to migrate from Xamarin to .NET MAUI, consider the following differences in the feature set of the Calendar control:
* Unlike the Xamarin Calendar, the .NET MAUI Calendar allows three selection modes—single, multiple, and range.
* Unlike the Xamarin Calendar, the .NET MAUI Calendar does not manage appointments. In Telerik UI for .NET MAUI, appointments will be handled by the .NET MAUI Scheduler control, which is planned for the R3 2023 release.
```